### ISSUE TITLE
Unable to reproduce results from scratch

### ISSUE THREAD

**Issue by zgpio** [imtiazziko/LaplacianShot#6]
Hello, I follow the description in the README：
> Alternatively to train the network on the base classes from scratch remove the "--evaluate " options in the following script. 

But the result I got at the end is far from the result you gave:
```
Meta Test: BEST
feature            UN               L2N            CL2N
GVP 1Shot       0.3235(0.0010)  0.3469(0.0011)  0.3877(0.0010)
GVP_5Shot       0.3192(0.0010)  0.3486(0.0011)  0.3933(0.0011)
```
Is it something I haven't noticed. Hope to get your reply and guidance

**Comment by imtiazziko** [comment]
This is weird. The model is not trained correctly then. Please check your training log and check the loss.
I think this can happen due to the pytorch version. This code was tested with Pytorch 1.2 and python 3.6.
Please check with the other pytorch version and see.

Thanks
